tox.ini

```diff
@@ -47,16 +48,21 @@ allowlist_externals =
 
 [testenv]
 setenv =
+    COV_CORE_SOURCE =
+    PYTEST_ADDOPTS = --numprocesses=logical --durations=10
+    coveralls: PYTEST_ADDOPTS = --numprocesses=logical --durations=10 --cov=xsdba --cov-report=term-missing
     PYTHONPATH = {toxinidir}
 passenv =
     COVERALLS_*
     GITHUB_*
 extras =
     dev
+    extras
 download = True
 install_command = python -m pip install --no-user {opts} {packages}
 deps =
     coveralls: coveralls>=4.0.1
+    numpy: numpy>=1.24.0,<2.0
     upstream: -r {toxinidir}/CI/requirements_upstream.txt
```

Comment:
in  a previous dependency file, it was simply 

numpy>=1.24

since I assume numba 0.57 supports numpy==2. So, it this "<2.0" necessary here?

Reply:
It's to verify that we still support older `numpy`. If we want to simply support `numpy` v2.0, we could change that in the `pyproject.toml`, but most of our base dependencies will be supporting `numpy` v1.0 until autumn 2025.

Reply:
See: https://scientific-python.org/specs/spec-0000/